core/src/main/scala/com/banno/kafka/consumer/ConsumerApi.scala

```diff
@@ -61,11 +61,11 @@ trait ConsumerApi[F[_], K, V] {
   def metrics: F[Map[MetricName, Metric]]
   def offsetsForTimes(
       timestampsToSearch: Map[TopicPartition, Long]
-  ): F[Map[TopicPartition, OffsetAndTimestamp]]
+  ): F[Map[TopicPartition, Option[OffsetAndTimestamp]]]
```

Comment:
For maps, maybe we should leave the value type without Option, since `map.get(key)` will return an Option?